src/main/java/com/puppycrawl/tools/checkstyle/checks/indentation/LineWrappingHandler.java

```diff
@@ -82,6 +82,8 @@ public static LineWrappingOptions ofBoolean(boolean val) {
         TokenTypes.RCURLY,
         TokenTypes.LITERAL_NEW,
         TokenTypes.ARRAY_INIT,
+        TokenTypes.LITERAL_DEFAULT,
+        TokenTypes.LITERAL_CASE,
```

Comment:
Core of the fix

Reply:
Does this fix affect regular (pre-java14) switch?

Reply:
As far as I understand switch **statements** (pre-java14) were not processed by LineWrappingHandler at all. From [Javadoc](https://github.com/checkstyle/checkstyle/blob/29e5e1210c6a292327a5a55b31b34180cd91f393/src/main/java/com/puppycrawl/tools/checkstyle/checks/indentation/LineWrappingHandler.java#L33-L35)
> This class checks line-wrapping into **definitions** and **expressions**.

 Sample scenarios covered by LineWrappingHandler **prior** to fix (processed part is highlighted, list if not definitive):
1. Package definition
 **package com.puppycrawl.tools.checkstyle.checks.indentation.indentation;**
2. Class(enum, interface...) definition
 **public class InputIndentationNew {**
3. Method(or constructor) definition
 **void test()** throws IOException  {
4. Variable definition
 **boolean test = true;**
5. if (expression? - not sure on category here)
**if (test)** {
6. Method call:
myfunc2 **(10, 10, 10)**
8. Annotation field:
**int value()** default 0;
9. Try-with-resources
        **try (**ZipFile zf = new ZipFile(zipFileName))** {
            
With java 14 switch expression can be part some constructions that are processed by LineWrappingHandler, e.g.:
```java
int i = switch("test") {
  case "first" -> 100;
  default -> -1;
};
methodCall(switch (line) {
  case "1" -> "0";
  default -> "1";
});
```